patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -560,7 +560,22 @@
              {
                  return false;
              }
-@@ -2011,6 +2167,11 @@
+@@ -1971,6 +2127,14 @@
+                 for (int j4 = j3; j4 < k3; ++j4)
+                 {
+                     IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos.func_181079_c(l3, i4, j4));
++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
++                    if(result != null) {
++                        if(result) {
```

Comment:
Please change the patch to this:
```java
 ++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
 ++                    if (result != null) {
 ++                        if (result) {
 ++                            blockpos$pooledmutableblockpos.func_185344_t();
 ++                            return true;
 ++                        }
 ++                        continue;
 ++                    }
```

`if` should have a space before the parenthesis.
`else` statements are not needed if the first part always returns.
When an `if` statement has braces, the `else` statement should have them too.

Reply:
How about this?

```java
 ++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
 ++                    if (result != null) {
 ++                        if (!result) continue;
 ++                        blockpos$pooledmutableblockpos.func_185344_t();
 ++                        return true;
 ++                    }
```

Reply:
That's good too.